app/services/payout/uphold_service.rb

```diff
@@ -44,15 +37,11 @@ def perform
 
       # Create potential payments for channel contributions
       @publisher.channels.verified.each do |channel|
-        probi = wallet.channel_balances[channel.details.channel_identifier].amount_probi # probi = balance - fee
-        fee_probi = wallet.channel_balances[channel.details.channel_identifier].fees_probi # fee = balance - probi
-        total_probi += probi
-
         potential_payments << PotentialPayment.new(
           payout_report_id: @payout_report&.id,
           name: "#{channel.publication_title}",
-          amount: "#{probi}",
-          fees: "#{fee_probi}",
+          amount: "0",
+          fees: "0",
```

Comment:
Maybe a null value here is better than 0.  We can achieve that by just deleting these lines and those columns won't be updated.

Reply:
There's a db restriction that discourages null values.